src/interpreter/s2s_reverse_mode_ad.jl

```diff
@@ -258,7 +258,7 @@ initial rdata directly into the statement, which is safe because it is always a
 """
 function reverse_data_ref_stmts(info::ADInfo)
     function make_ref_stmt(id, P)
-        ref_type = Base.RefValue{P<:Type ? NoRData : zero_like_rdata_type(P)}
+        ref_type = Base.RefValue{P <: Type ? NoRData : zero_like_rdata_type(P)}
```

Comment:
Let's avoid unnecessary styling changes in PRs. Same comments for a few other places. Please revert these styling changes.